### ISSUE TITLE
No Google Signup Popup at Udemy

### ISSUE THREAD

**Issue by MitchEspr** [brave/brave-browser#15211]
## Description 
When I try to login at Udemy.com to my account using the _Continue with Google_ option, the signup popup window doesn't show and the _Continue with Google_ button greys out. Brave Shields are turned OFF.

## Steps to Reproduce

   1.  Login at Udemy.com
   2.  Click on _Continue with Google_ button

## Actual result:

![image](https://user-images.githubusercontent.com/73789128/114211546-1188fd80-9961-11eb-8089-0816f44f4a9b.png)

## Expected result:
![image](https://user-images.githubusercontent.com/73789128/114211642-2cf40880-9961-11eb-97e9-cac3e7d440bf.png)

## Reproduces how often: 

Easily reproduced.

## Brave version (brave://version info)

Brave | 1.22.71 Chromium: 89.0.4389.114 (Official Build) (64-bit)
Revision | 1ea76e193b4fadb723bfea2a19a66c93a1bc0ca6-refs/branch-heads/4389@{#1616}
OS | Windows 10 OS Version 2004 (Build 19041.867)

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? not verified
- Can you reproduce this issue with the nightly channel? not verified

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? no
- Does the issue resolve itself when disabling Brave Rewards? no
- Is the issue reproducible on the latest version of Chrome? no

**Comment by bsclifton** [comment]
@MitchEspr I believe this is expected behavior. Can you please try enabling `Allow Google login buttons on third party sites` in `brave://settings/socialBlocking`?

If for some reason that doesn't work, you can also try to toggle `Allow Google login for extensions` under `brave://settings/extensions`

**Comment by MitchEspr** [comment]
@bsclifton Thanks for the response. Sorry for not mentioning it in my earlier post, but both settings were already enabled when I encountered the issue for the first time. Turning the settings off and re-enabling them does not resolve the issue.

**Comment by rebron** [comment]
@MitchEspr Can you see if on the latest 1.26.77 this is working for you now?
<img width="1363" alt="Screen Shot 2021-07-20 at 10 12 37 AM" src="https://user-images.githubusercontent.com/5951041/126366997-5e03f876-0ac0-4ad7-886f-69a44eb1427c.png">

I was able to see the Google sign-up in either a new login to Google or already logged-in to Google state.

**Comment by MitchEspr** [comment]
@rebron Unfortunately, the problem for me remains as stated originally.

**Comment by rebron** [comment]
@MitchEspr Do you have an extension installed that you think might conflict or block here? Thanks for bringing up the initial issue and helping us troubleshoot. If you can, can you create a new profile and see if it's still not displaying with that new profile? 

Possibly clearing cookies from Udemy might solve it as well.

**Comment by MitchEspr** [comment]
@rebron Thanks for the suggestions! I started by creating a new profile, which indeed solved the issue. Since the new profile didn't have any extensions installed, I figured it might indeed have something to do with a conflicting extension, so I started to disenable all extensions on my original profile, which also solved the issue on my original profile. I then proceeded to reenable the extension one by one to see which one was causing the problem, but once I had all original extensions enabled again, the problem still didn't reappear. So I didn't manage to figure out what was causing the problem, but going through this process did resolve the issue for me.
